ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -210,7 +210,7 @@ def build(objdir, target_name, kind_name=None,
 
     if os.path.isdir(bindgen):
         bindgen = ["cargo", "run", "--manifest-path",
-                   os.path.join(bindgen, "Cargo.toml"), "--features", "llvm_stable", "--"]
+                   os.path.join(bindgen, "Cargo.toml"), "--"]
```

Comment:
Yeah we can't do this, because llvm_stable is the thing that's supposed to allow us to build with llvm38, and we need that.